todo-backend/README.md

```diff
@@ -0,0 +1,67 @@
+# Todo-Backend
+
+[Todo-Backend](https://www.todobackend.com/) is a typical example to showcase backend tech stacks. It defines a simple web API for managing a Todo List.
+
+This sample is an implementation of a Todo-Backend with Binaris functions.
+
+# How does it work?
+
+1. A public function exposes REST API to the outside world
+1. A private function uses a Redis Cache instance for the data storage purpose
+
+# Deploy me!
+
+## Get some API Keys
+1. [Binaris API Key](https://binaris.com/)
+1. A Redis instance. A free, managed one is available from [Redis Labs](www.redislabs.com)
+
+## Deploy
+```bash
+# Install the Binaris CLI
+$ npm install -g binaris
+
+# Set your API key in ~/.binaris.yml
+$ bn login
+
+# Clone this repo
+$ git clone git@github.com:binaris/functions-examples.git
+$ cd functions-examples/todo-backend
+
+# Set the environment variables for deployment
+$ export BINARIS_ACCOUNT_ID={your-binaris-account-id}
+$ export BINARIS_API_KEY={your-binaris-api-key}
+$ export REDIS_HOST={your-redis-host}
+$ export REDIS_HOST={your-redis-port}
+$ export REDIS_PASSWORD={your-redis-password}
+
+# Install the dependencies and deploy the API function
+$ cd TodoBackend
+$ npm install
+$ bn deploy public_TodoBackend
+$ cd ..
+
+# Install the dependencies and deploy the storage function
+$ cd TodoRedis
+$ npm install
+$ bn deploy TodoRedis
+```
```

Comment:
we prefer to have one binaris.yml in the root directory of the project and have the functions code in another. Take a look at how the code is structured on https://github.com/binaris/functions-examples/tree/master/whiteboard. You will have to change the file structure.

Reply:
Yes, I started with the single-folder setup, but I found no way to add the second function with a CLI command. Since this example is for the tutorial, I want it to be exactly the end result of the tutorial.
Or should I describe the manual steps of adding the second function in the tutorial?

Reply:
yes I think it is better that way and since your tutorial is pretty advanced (testing, CORS etc.) we might as well show them there is a better way